src/phoenix/db/migrations/versions/cf03bd6bae1d_init.py

```diff
@@ -49,7 +49,13 @@ def upgrade() -> None:
     op.create_table(
         "traces",
         sa.Column("id", sa.Integer, primary_key=True),
-        sa.Column("project_rowid", sa.Integer, sa.ForeignKey("projects.id"), nullable=False),
+        sa.Column(
+            "project_rowid",
+            sa.Integer,
+            sa.ForeignKey("projects.id", ondelete="CASCADE"),
+            nullable=False,
+            index=True,
```

Comment:
Are these indexes anonymous? Do we need to explicitly name them in order to run migrations on them?

Reply:
They're named automatically.

<img width="208" alt="Screenshot 2024-04-12 at 10 29 35 AM" src="https://github.com/Arize-ai/phoenix/assets/80478925/34cea3d2-6105-4e04-a4cd-44904c849587">